grpcutil/metadata_test.go

```diff
@@ -7,24 +7,25 @@ import (
 
 	"github.com/stretchr/testify/require"
 	"google.golang.org/grpc/metadata"
-
-	"github.com/grafana/dskit/httpgrpc"
 )
 
 func TestAppendMessageSizeToOutgoingContext(t *testing.T) {
 	ctx := context.Background()
 
-	req := &httpgrpc.HTTPRequest{
```

Comment:
This has been replaced with `fakeSizer` to avoid a circular dependency. The parameter of `AppendMessageSizeToOutgoingContext` should be any implementation of the `Sizer` interface, and `httpgrpc.HTTPRequest` was just one of them.